pkg/integrations/blackbox_exporter/blackbox.go

```diff
@@ -0,0 +1,19 @@
+package blackbox_exporter
+
+import (
+	"net/http"
+
+	"github.com/go-kit/log"
+	blackbox_config "github.com/prometheus/blackbox_exporter/config"
+	"github.com/prometheus/blackbox_exporter/prober"
+)
+
+type blackboxHandler struct {
+	cfg     *Config
+	modules *blackbox_config.Config
+	log     log.Logger
+}
+
+func (bbh *blackboxHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	prober.Handler(w, r, bbh.modules, bbh.log, &prober.ResultHistory{}, 0.5, nil)
```

Comment:
What is the `0.5` magic number?

Reply:
Taken from this: https://github.com/prometheus/blackbox_exporter/blob/master/main.go#L61